torch/ao/quantization/experimental/quantizer.py

```diff
@@ -1,12 +1,58 @@
+import torch
 from torch import Tensor
+from torch.ao.quantization.experimental.observer import APoTObserver
+from torch.ao.quantization.experimental.apot_utils import float_to_apot
 
 # class to store APoT quantizer
 # implements quantize and dequantize
 # and stores all quantization parameters
 class APoTQuantizer():
-    @staticmethod
-    def quantize_APoT(tensor2quantize: Tensor) -> Tensor:
-        raise NotImplementedError
+    b: int
+    k: int
+    n: int
+    signed: bool
+    quantization_levels: torch.Tensor
+    level_indices: torch.Tensor
+    data: torch.Tensor
```

Comment:
if we want to be aligned with C++ api, data should probably be stored in a quantized tensor I think, quantizer can store all the quantization parameters

Reply:
I think we should make this change. missed this earlier. data isn't an attribute of the quantizer